python/pyproject.toml

```diff
@@ -45,6 +45,11 @@ requires = [
 build-backend = "mesonpy"
 
 [tool.meson-python.args]
+# Consistent version for zstd, rather than attempting to delocate whatever
```

Comment:
When you say delocate are you just referring to just to macOS or does this affect all platforms?

Reply:
Ah ok. I can take a look too. 

Thinking out loud, we might need to be careful if it's the wheel build itself installing the libraries (because the subproject meson configurations instruct the build system to do so) or if it's the wheel repair jobs that are bundling improper libraries. I'm guessing it's the former. 

I think we could add the following to pyproject.toml to prevent the former issue:

[tool.meson-python.args]
install = ['--skip-subprojects']

Or possibly even statically link the zstd subproject (?)